FrontEnd/SignUpPage/src/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -0,0 +1,267 @@
+import { useForm } from 'react-hook-form';
+import styles from './SignUpFormContent.module.css'
+
+function SignUpFormContentComponent() {
+
+    const errorMessageTemplates = {
+        'required': 'Обов’язкове поле',
+        'email': 'invalid email, should contain @ and .xx',
+        'password': 'Пароль не відповідає вимогам',
+        'confirmPassword': "Паролі не збігаються",
+        'checkboxes': 'Будь ласка, оберіть кого ви представляєте'
+    }
+
+    const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+    const passwordPattern = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{8,20}$/;
+
+
+    const {
+        register,unregister,
+        handleSubmit,
+        formState: { isValid, isSubmitted, errors },
+        watch, getValues, setValue
+    } = useForm();
+
+
+    const onSubmit = (event) => {
+        // event.preventDefault();
+        console.log(isValid)
+        if (isValid) {
+            console.log("Successfully submitted: ", isSubmitted)
+            console.log("Values: ", getValues())
+        }
```

Comment:
I assume that API is not completed if we fire console log here.
Its better to left `TODO` here if we want to come back instead of many logs.